packages/mvc.php

```diff
@@ -0,0 +1,343 @@
+<?php
+/**
+ * MVC exporter tool.
+ *
+ * @copyright 2009-2018 Vanilla Forums Inc.
+ * @license GNU GPL2
+ * @package VanillaPorter
+ * @see functions.commandline.php for command line usage.
+ */
+
+$supported['mvc'] = array('name' => 'mvc', 'prefix' => 'mvc_');
+$supported['mvc']['features'] = array(
+    'Comments' => 1,
+    'Discussions' => 1,
+    'Users' => 1,
+    'Badge,' => 1,
+    'Categories' => 1,
+    'Roles' => 1,
+    'Avatars' => 1,
+    'Attachments' => 0,
+    'Signatures' => 1,
+    'Tags' => 1,
+);
+
+class MVC extends ExportController {
+    /**
+     * You can use this to require certain tables and columns be present.
+     *
+     * @var array Required tables => columns
+     */
+    protected $sourceTables = array(
+        'MembershipUser' => array(),
+    );
+
+    /**
+     * Main export process.
+     *
+     * @param ExportModel $ex
+     * @see $_Structures in ExportModel for allowed destination tables & columns.
+     */
+    public function forumExport($ex)
+    {
+
+        $characterSet = $ex->getCharacterSet('posts');
+        if ($characterSet) {
+            $ex->characterSet = $characterSet;
+        }
+
+        // Reiterate the platform name here to be included in the porter file header.
+        $ex->beginExport('', 'MVC');
+
+        $structures = $ex->structures();
+        $structures['Badge'] = array(
+            'BadgeID' => 'int',
+            'Name' => 'varchar(64)',
+            'Slug' => 'varchar(32)',
+            'Type' => 'varchar(20)',
+            'Body' => 'Text',
+            'Photo' => 'varchar(255)',
+            'Points' => 'int',
+            'Active' => 'tinyint',
+            'Visible' => 'tinyint',
+            'Secret' => 'tinyint',
+            'CanDelete' => 'tinyint',
+            'DateInserted' => 'datetime',
+            'DateUpdated' => 'datetime',
+            'InsertUserID' => 'int',
+            'UpdateUser' => 'int',
+            'CountRecipients' => 'int',
+            'Threshold' => 'int',
+            'Class' => 'varchar(20)',
+            'Level' => 'smallint',
+            'Attributes' => 'text'
+        );
+
+        $structures['UserBadge'] = array(
+            'UserID' => 'int',
+            'BadgeID' => 'int',
+            'Attributes' => 'text',
+            'Reason' => 'varchar(255)',
+            'ShowReason' => 'tinyint',
+            'DateRequested' => 'datetime',
+            'RequestReason' => 'varchar(255)',
+            'Declined' => 'tinyint',
+            'Count' => 'int',
+            "DateCompleted" => 'datetime',
+            'DateInserted' => 'datetime',
+            'InsertUserID' => 'int'
+        );
+
+        $ex->structures($structures);
+
+        // Users.
+        $user_Map = array();
```

Comment:
Because the 3rd parameter of the `exportTable` function in the `ExportModel` class defaults to an empty array, you don't need to declare an array in your package if you don't need it.

https://github.com/vanilla/porter/blob/8b4a0b906d286ae8b48f03a1fdb1620c23cbea27/class.exportmodel.php#L257

```suggestion
```